### PR TITLE
Add batch consumption of messages

### PR DESCRIPTION
**Pull request by nguyenuy**
Addressing issue #66 with this commit. Also added appropriate test coverage. Please take a look

### CODE REVIEW

File: embedded-kafka/src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala [hunk 1]
```diff
@@ -271,6 +282,69 @@ sealed trait EmbeddedKafkaSupport {
...
+    * @param deserializer an implicit [[org.apache.kafka.common.serialization.Deserializer]] for the type [[T]]
+    * @return the first message consumed from the given topic, with a type [[T]]
+    * @throws TimeoutException          if unable to consume a message within 5 seconds
+    * @throws KafkaUnavailableException if unable to connect to Kafka
+    */
+  def consumeNumberMessagesFrom[T](topic: String, number: Int, autoCommit: Boolean = false)(
+    implicit config: EmbeddedKafkaConfig,
+    deserializer: Deserializer[T]): List[T] = {
+
+    import scala.collection.JavaConversions._
```
Comment (manub) [inline embedded-kafka/src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala]: `JavaConversions` is deprecated in 2.12. Could you please use `JavaConverters` instead?

**Comment by nguyenuy** [comment]
@manub added import scala.collection.JavaConverters._ within that specific function. Didn't want to do too much changing elsewhere

**Comment by nguyenuy** [comment]
Is there anything else you need for this PR?

**Comment by manub** [comment]
Sorry I didn't have time to carry on reviewing! LGTM, will merge.

**Comment by manub** [comment]
I will release a new patch version shortly.

**Comment by manub** [comment]
Released in v0.11.1

**Pull request merged**